x-pack/plugins/maps/public/shared/layers/tooltips/es_tooltip_property.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { buildPhraseFilter } from '@kbn/es-query';
+import { filterBarQueryFilter } from '../../../kibana_services';
+import { TooltipProperty } from './tooltip_property';
+import _ from 'lodash';
+
+export class ESTooltipProperty extends TooltipProperty {
+
+  constructor(propertyName, rawValue, indexPattern) {
+    super(propertyName, rawValue);
+    this._indexPattern = indexPattern;
+  }
+
+  getHtmlDisplayValue() {
+
+    if (typeof this._rawValue === 'undefined') {
+      return '-';
+    }
+
+    const field = this._indexPattern.fields.byName[this._propertyName];
+    if (!field) {
+      return _.escape(this._rawValue);
+    }
+    const htmlConverter = field.format.getConverterFor('html');
+    return  (htmlConverter) ? htmlConverter(this._rawValue) :
```

Comment:
extra space between `return  (htmlConverter)`. Parentheses not needed around `(htmlConverter)`